pkg/apis/core/validation/validation.go

```diff
@@ -4939,3 +4939,13 @@ func ValidateCIDR(cidr string) (*net.IPNet, error) {
 	}
 	return net, nil
 }
+
+func IsDecremented(update, old *int32) bool {
```

Comment:
PLEASE add comments for public functions

Reply:
agree. thought this was a straight move and didn't notice it was not exported previously.

Reply:
OK,I have add comments  in new pr